nbextensions/bulkImportCell/tabs/fileTypePanel.js

```diff
@@ -167,17 +168,19 @@ define(['bluebird', 'common/ui', 'common/html', 'common/events'], (Promise, UI,
                     ui.getElement(key).classList.remove(selected);
                 }
                 ui.getElement(`${key}.icon`).classList.remove(...completeIcon, ...incompleteIcon);
-                const icon = state.completed[key] ? completeIcon : incompleteIcon;
-                ui.getElement(`${key}.icon`).classList.add(...icon);
-                // if there's a warning, and the fileType doesn't already have a warning icon,
-                // make one and insert it.
-                const warningIconElem = ui.getElement(`${key}.warning-icon`);
-                if (state.warnings.has(key) && !warningIconElem) {
-                    addWarningIcon(key);
-                }
-                // if the state has changed so that the warning icon shouldn't be there, remove it.
-                else if (!state.warnings.has(key) && warningIconElem) {
-                    warningIconElem.remove();
+                if (!viewOnly) {
```

Comment:
The icons get set when updating the state, so just skip that bit in view only mode.
State updates also change the button highlighting to the active file type, so that needs to stay.